src/Analyzers/Analyzers/test/CompilationFeatureDetectorTest.cs

```diff
@@ -29,7 +29,6 @@ public async Task DetectFeaturesAsync_FindsNoFeatures()
         }
 
         [Theory]
-        [InlineData(nameof(StartupWithUseSignalR))]
```

Comment:
We don't have a provision for building the analyzer tests against older runtimes, though it might be feasible with TFM pivots. Doesn't feel super necessary though.